### PR TITLE
Migration to rename autoplay_enabled to tts_autoplay_enabled

### PR DESCRIPTION
**Pull request by indhu-gunda**
migration for renaming autoplay_enabled to tts_autoplay_enabled for clarity and the resulting model updates & schema change

### CODE REVIEW

File: dashboard/db/schema.rb [hunk 1]
```diff
@@ -332,7 +332,7 @@
     t.index ["level_group_level_id"], name: "index_contained_levels_on_level_group_level_id", using: :btree
   end
 
-  create_table "course_offerings", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+  create_table "course_offerings", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci" do |t|
```
Comment (clareconstantine) [inline dashboard/db/schema.rb:335]: this looks like an unrelated change - also to the `course_versions` and `foorm_misc_surveys` tables

**Review by clareconstantine — COMMENTED** [review]
Looks like a couple unintentional changes are in the schema.rb, but otherwise looks good!

File: dashboard/db/schema.rb (hunk 1 above)
Comment (dmcavoy) [reply dashboard/db/schema.rb:335]: @clareconstantine whats the best way to fix this?

Comment (clareconstantine) [reply dashboard/db/schema.rb:335]: running `rails db:migrate` should regenerate the schema.rb - if that doesn't fix it, then I think it means the local db has changes in it that aren't represented by migrations, and we could probably write a migration locally to fix that, or @indhu-gunda could drop their db and recreate it. But hopefully rerunning the existing migrations is all that's needed!

**Comment by maureensturgeon** [comment]
Work picked up in https://github.com/code-dot-org/code-dot-org/pull/38932
